editor/code_editor.cpp

```diff
@@ -1020,8 +1020,13 @@ void CodeTextEditor::_line_col_changed() {
 
 void CodeTextEditor::_text_changed() {
 
-	code_complete_timer->start();
-	idle->start();
+	String next_char;
+	next_char = text_editor->get_line(text_editor->cursor_get_line()).substr(text_editor->cursor_get_column(), 1);
```

Comment:
No need to use two statements for this, you can do `String next_char = text_editor->...` directly.

Reply:
thanks, i'll update. This is just a little hack to demonstrate how, i think, the behaviour of the popup should be. I don't know c++ to do a proper change. I've update the [bug(behaviour)](https://github.com/godotengine/godot/issues/10695) report with some animated gifs, the previous one was unclear.